code/Master.py

```diff
@@ -731,13 +745,29 @@ def generate_buffered_population(paths, param):
 
     array2raster(paths["BUFFER"], GeoRef["RasterOrigin"], GeoRef["pixelWidth"], GeoRef["pixelHeight"],
                  A_notPopulated)
-    create_json(paths["BUFFER"], param, ["region_name", "landuse", "WindOn", "Crd_all", "res_desired", "GeoRef"], paths, ["LU", "BUFFER"])
+    create_json(paths["BUFFER"], param, ["region_name", "landuse", "WindOn", "Crd_all", "res_desired", "GeoRef"], paths,
+                ["LU", "BUFFER"])
     print("files saved: " + paths["BUFFER"])
     timecheck('End')
 
 
-def generate_wind_correction(paths, param):
+def generate_area(paths, param):
+    timecheck('Start')
+    Crd_all = param["Crd_all"]
+    n_high = param["n_high"]
+    res_desired = param["res_desired"]
+
+    # Calculate available Area
+    A_area = calc_areas(Crd_all, n_high, res_desired)
```

Comment:
I was thinking to completely move the content of "calc_areas" into "generate_area"